documentation/src/main/markdown/gettingstarted/terms/scans.md

```diff
@@ -1,4 +1,5 @@
 # Scans and projects
 
-[solution_name] scans are mapped to one project. A project version can have more than one scan mapped to it, which enables the mapping of multiple separate folders and their scan results into one aggregated project version.
+A project is the base unit in [blackduck_product_name] and [solution_name]. A project can be both a stand-alone development project and part of another project. For example, Apache Tomcat is a project but it may also be part of other, larger projects, that make use of its functionality. 
```

Comment:
both?

A project can be a stand-alone development project or a part of another project.

Reply:
I'd say it is technically both. Like the example with Tomcat, it is its own project which could be scanned, and then be part of other projects, such as when used as library by Detect. (Though it would be one "component" in both cases)